semantic-release/action.yaml

```diff
@@ -124,7 +124,7 @@ runs:
       shell: bash
       run: |
         cat <<EOF > .releaserc.yaml
-        branches: ["${GITHUB_REF#refs/*/}"]
+        branches: ["${GITHUB_REF#refs/*/}","${{ github.event.repository.default_branch }}"]
```

Comment:
Der Default-Branch ist aber nicht immer der Release-Branch. Z.b. wenn wir einen `develop` branch haben.

Reply:
Das hier ist nur ein dry run, um die version zu erhalten. Da es ein branch sein muss, bietet sich das default branch an. Es wird am ende eh der branch in `$GITHUB_REF}` gewählt